fixtures/invalid/digitization_bag/4c50bbba-5698-4add-893a-27be921f0345.json

```diff
@@ -5,43 +5,34 @@
         {
```

Comment:
I would copy and paste a valid rights statement in here, presuming that this is testing something other than whether or not the embedded rights statement is valid. I think we should also rename this fixture so it's more obvious what exactly this is invalidating.